### ISSUE TITLE
NVDA file will not open

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2198]
**Reported by currydj1981 on 2012-03-24 18:09**
When attempting to open file the following command appears:

Errors occurred

The logfile 'C:\Program Files (x86)\NVDA\nvda.exe.log' could not be opened: [13](Errno) Permission denied: 'C:\Program Files (x86)\NVDA\nvda.exe.log'

**Comment by dkager** [comment]
Looks abandoned.
Note that even a user account with limited privileges should be able to read the log file, regardless of whether the user or the Administrator owns it.

**Comment by jcsteh** [comment]
This was referring to a different log file that gets created (I think as part of the py2exe bootstrapper) when NVDA fails very early in startup. It can't create the log file when not running with admin privileges because it can't write to Program Files. As you say, there's insufficient info and no further info from reporter, so closing.
